griptape/utils/chat.py

```diff
@@ -15,23 +16,37 @@ class Chat:
     intro_text: Optional[str] = field(default=None, kw_only=True)
     prompt_prefix: str = field(default="Q: ", kw_only=True)
     response_prefix: str = field(default="A: ", kw_only=True)
+    streaming_output_fn: Callable[[str], None] = field(
+        default=lambda x: print(x, end=""), kw_only=True
+    )
     output_fn: Callable[[str], None] = field(
         default=Factory(lambda: print), kw_only=True
     )
 
-    def start(self) -> None:
+    def start(self, initial_turn: str = None) -> None:
+        if initial_turn:
+            self.output_fn(initial_turn)
+
         if self.intro_text:
             self.output_fn(self.intro_text)
         while True:
             question = input(self.prompt_prefix)
 
             if question.lower() in self.exit_keywords:
                 self.output_fn(self.exiting_text)
-
                 break
             else:
                 self.output_fn(self.processing_text)
 
-            self.output_fn(
-                f"{self.response_prefix}{self.structure.run(question).output.to_text()}"
-            )
+            if self.structure.stream:
+                first_chunk: bool = True
+                for chunk in Stream(self.structure).run(question):
+                    if first_chunk:
+                        self.streaming_output_fn(f"{self.response_prefix}")
+                        first_chunk = False
+                    self.streaming_output_fn(chunk.value)
+                self.streaming_output_fn("\n")
+            else:
+                self.output_fn(
+                    f"{self.response_prefix}{self.structure.run(question).output.to_text()}"
```

Comment:
I just changed your branch to be based off `dev`. And in this branch we changed the return value of `run` to be `Structure`. Can you please update `output.to_text()` to `output_task.output.to_text()`?